src/Yoti/YotiClient.php

```diff
@@ -133,8 +134,8 @@ public function getOutcome()
      *
      * @return \Yoti\ActivityDetails
      *
-     * @throws \Exception
-     * @throws \Yoti\Exception\ActivityDetailsException
+     * @throws AttributeException
```

Comment:
I don't think we want it to throw AttributeException.
If there is a problem parsing a _DocumentDetails_ attribute, we'd want it to log a _warning_ (or the PHP equivalent) and continue as if the attribute was never there.

Reply:
Yes the exception is fine, I will log the waring in `ActivityDetails::getYotiAttribute` inside the catch.